css-positioning/relative.css

```diff
@@ -0,0 +1,36 @@
+.pokemon {
+  width: 900px;
+  margin: 0 auto;
+  text-align: center;
+}
+
+.pokemon > img {
+  width: 150px;
+
+  /* your code here */
```

Comment:
You can apply `position: relative` here so that you don't need to repeat it in each rule.